docs/user-guide/secrets/index.md

```diff
@@ -267,7 +318,7 @@ spec:
   restartPolicy: Never
 ```
 
-##### Consuming Secret Values from Environment Variables
+**Consuming Secret Values from Environment Variables**
```

Comment:
As you requested to remove all 5-level headings, what other options besides asterisks is available?